roles/mesos/defaults/main.yml

```diff
@@ -8,7 +8,8 @@ mesos_agent_package: "mesos-agent-{{ mesos_version }}"
 mesos_mode: follower
 mesos_log_dir: /var/log/mesos
 mesos_logging_level: WARNING
-mesos_work_dir: /var/lib/mesos
+mesos_work_dir: /mnt/mesos
```

Comment:
Why not have mesos and docker also use mantldata? Which components should and shouldn't use this new volume, and why?